scripts/vim-patch.sh

```diff
@@ -7,7 +7,7 @@ set -p
 
 # Ensure that the user has a bash that supports -A
 if [[ "${BASH_VERSINFO[0]}" -lt 4  ]]; then
-  echo "This script requires bash version 3 or later (you have ${BASH_VERSION})." >&2
+  echo "This script requires bash version 4 or later (you have ${BASH_VERSION})." >&2
```

Comment:
i [noticed that](https://github.com/neovim/neovim/pull/12081#discussion_r415506519) too, thanks :)

while we're here let's make the message less verbose:

```suggestion
  >&2 echo "error: script requires bash 4+ (you have ${BASH_VERSION})."
```

Reply:
👍 committed the suggestion via GitHub. The `^M` carriage return  got replaced by a new line as well, which I guess is more ideal